plugins/inventory/vmware_host_inventory.py

```diff
@@ -202,14 +202,18 @@ def parse(self, inventory, loader, path, cache=True):
         # set _options from config data
         self._consume_options(config_data)
 
+        username = self.get_option("username")
         password = self.get_option("password")
 
+        if isinstance(username, AnsibleVaultEncryptedUnicode):
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Add a note in `username` and `password` description that this supports Ansible Vault Encrypted String.